Update IPA Installation Techniques and Tools (by @NVISOSecurity) (#3100)
OWASP/owasp-mastg: techniques/ios/MASTG-TECH-0092.md

[15:22:03]: fastlane.tools finished successfully 🎉
```

> After setting this set up, you only need to change the path in the `Fastfile` for the IPA you want to resign and execute the command again.

Reviewer: ```suggestion
Once this is set up, all you need to do is change the path in the `Fastfile' for the IPA you want to resign and run the command again.
```
